packages/gatsby-theme-api-docs/src/hooks/use-api-resources.js

```diff
@@ -28,6 +28,11 @@ export const useApiResources = () => {
             applicationjson {
               type
               builtinType
+              examples {
+                key
+                strict
```

Comment:
```suggestion
```
the "strict" information is not necessary for the frontend, it's just indicating the RAML parser and validators that the example is supposed to be validated against the schema or not (for example allowing for pseudo-code examples that are not validated)